cit3.0-web/src/components/Page/ReviewOpportunity/ReviewOpportunity.js

```diff
@@ -64,16 +61,14 @@ const ReviewOpportunity = ({ location }) => {
       <ReviewAndSubmitCallout
         submitOpportunity={() => handleSubmitOpportunity()}
         cancelOpportunity={() => confirmCancel()}
-        prevRoute="/addOpportunity/propDetails2"
+        prevRoute="/addOpportunity/addional-details"
       />
-      <OpportunityView data={location.state} view="all" />
+      <OpportunityView view="all" />
     </div>
   );
 };
 
-ReviewOpportunity.propTypes = {
-  location: PropTypes.shape().isRequired,
-};
+ReviewOpportunity.propTypes = {};
```

Comment:
Should we just remove these if we aren't using them?

Reply:
Yeah, I suppose, as long as lint is happy